### PR TITLE
UIDEXP-35: Populate hrId fields

### PR DESCRIPTION
**Pull request by Yuliia-Dovhal**
## Purpose:
Populate Job Id in "Running jobs" and "Logs" components on the landing page.

[UIDEXP-35](https://issues.folio.org/browse/UIDEXP-35)

## Screenshot:
Screenshot 2020-03-20 at 21 51 02

### CODE REVIEW

File: src/components/Jobs/RunningJobs/JobDetails/JobDetails.js [hunk 1]
```diff
@@ -24,14 +24,17 @@ const formatTime = dateStr => {
...
-// TODO: remove defaults for jobProfileInfo, hrId and runBy once backend is in place
 const JobDetails = props => {
   const { job } = props;
 
   const {
     jobProfileInfo,
     exportedFiles,
-    runBy,
+    hrId,
+    runBy: {
```
Comment (VictorSoroka1) [inline src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]: Let's not mix runBy feature in this PR. Create another PR for it, or add all needed changes to make runBy field also visible.

File: src/components/Jobs/RunningJobs/RunningJobs.js [hunk 2]
```diff
@@ -5,18 +5,26 @@ import {
   JobsListAccordion,
 } from '@folio/stripes-data-transfer-components';
 
+import { JOB_EXECUTION_STATUSES } from '../../../utils/constants';
 import ItemFormatter from './ItemFormatter';
 import { DataFetcherContext } from '../../../contexts/DataFetcherContext';
 
+const prepareJobsData = jobs => {
```
Comment (VictorSoroka1) [inline src/components/Jobs/RunningJobs/RunningJobs.js]: No need for changes in this file.

File: src/components/Jobs/RunningJobs/JobDetails/JobDetails.js (hunk 1 above)
Comment (Yuliia-Dovhal) [reply src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]: Yeah, good catch.

File: src/components/Jobs/RunningJobs/RunningJobs.js (hunk 2 above)
Comment (Yuliia-Dovhal) [reply src/components/Jobs/RunningJobs/RunningJobs.js]: Okay, I'll revert. Thanks!

File: src/components/Jobs/RunningJobs/JobDetails/JobDetails.js [hunk 3]
```diff
@@ -42,15 +42,17 @@ const JobDetails = props => {
...
+        <span data-test-running-job-triggered-by>
+          {runBy && (
+            <span data-test-running-job-triggered-by>
+              <FormattedMessage
+                id="ui-data-export.triggeredBy"
+                values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
+              />
+            </span>
+          )}
+        </span>
```
Comment (Yuliia-Dovhal) [inline src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]: @VictorSoroka1 
That changes was before my last commit, so I decided just revert my changes and leave this ones

File: src/components/Jobs/RunningJobs/JobDetails/JobDetails.js [hunk 4]
```diff
@@ -42,15 +42,17 @@ const JobDetails = props => {
...
-        {runBy && (
-          <span data-test-running-job-triggered-by>
-            <FormattedMessage
-              id="ui-data-export.triggeredBy"
-              values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
-            />
-          </span>
-        )}
+        <span data-test-running-job-hr-id>{hrId}</span>
+        <span data-test-running-job-triggered-by>
```
Comment (VictorSoroka1) [inline src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]:
```suggestion
          {runBy && (
            <span data-test-running-job-triggered-by>
              <FormattedMessage
                id="ui-data-export.triggeredBy"
                values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
              />
            </span>
          )}
```

File: src/components/Jobs/RunningJobs/JobDetails/JobDetails.js [hunk 5]
```diff
@@ -42,14 +42,14 @@ const JobDetails = props => {
...
-        <span data-test-running-job-hr-id>{get(job, 'hrId', '')}</span>
+        <span data-test-running-job-hr-id>{hrId}</span>
         {runBy && (
-          <span data-test-running-job-triggered-by>
-            <FormattedMessage
-              id="ui-data-export.triggeredBy"
-              values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
-            />
-          </span>
+        <span data-test-running-job-triggered-by>
```
Comment (VictorSoroka1) [inline src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]: Indentation is needed:)

Comment (Yuliia-Dovhal) [reply src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]: Before 
{runBy ... ?

My esLint not show error on this, sorry

Comment (VictorSoroka1) [reply src/components/Jobs/RunningJobs/JobDetails/JobDetails.js]: I pushed, lint did not seem to bother.

**Pull request merged**